README.md

```diff
@@ -1,250 +1,326 @@
+
+
+:: Images  
+
+firmwares/            - Folder where the final images end up 
 
 ```
-git clone https://github.com/freifunk-berlin/firmware.git
-cd firmware
+
+
+### Building all firmware images
+
+To get the source and build the firmware locally use:
+
+``` 
+git clone https://github.com/freifunk-berlin/firmware.git 
+cd firmware 
 make
 ```
 
-The build will take some time. You can improve the build time with [build options](https://openwrt.org/docs/guide-developer/build-system/use-buildsystem)
+The build will take some time. You can improve the build time with [build
+options](https://openwrt.org/docs/guide-developer/build-system/use-buildsystem)
 such as `-j <number of cores>`. `V=s` will give more verbose error messages.
 
 An internet connection is required during the build process. A good internet
 connection can improve the build time.
 
 You need approximately 10GB of space for the build.
 
-### Directory Layout
-
-You can find the actual firmware images generated by the ImageBuilder (and the ImageBuilder itself)
-in `firmwares`. The layout looks like the following:
 
+### Directory Layout of `firmwares/`
+
+You can find the actual firmware images generated by the ImageBuilder (and the
+ImageBuilder itself) in `firmwares/`. The layout looks like the following:
+
+``` 
+firmwares/ 
+  TARGET/ 
+    backbone/ 
+      images.. 
+    default/
+      images..
+    ...
+    OpenWrt-ImageBuilder-....tar.xz 
+    OpenWrt-SDK-....tar.xz
+    initrd/ 
+      images..
+    packages/ 
+      packages/<ARCH>
+        base/*.ipk 
+        luci/*.ipk
+        packages/*.ipk
+        packages_berlin/*.ipk
+        routing/*.ipk
+      targets/MAINTARGET/SUBTARGET/packages/
+          *.ipk
 ```
-firmwares/
-    TARGET/
-        backbone/
-           images..
-        default/
-           images..
-        ...
-        OpenWrt-ImageBuilder-....tar.xz
-        OpenWrt-SDK-....tar.xz
-        initrd/
-           images..
-        packages/
-           packages/<ARCH>
-              base/*.ipk
-              luci/*.ipk
-              packages/*.ipk
-              packages_berlin/*.ipk
-              routing/*.ipk
-           targets/MAINTARGET/SUBTARGET/packages/
-              *.ipk
-```
 
-As you notice there are several different image variants ("backbone", "default", etc.).
-These different *packages lists* are defined in `packages/`.
-See the "Features" section above for a description of the purpose of each package list.
-With the "OpenWrt-Imagebuilder" you can assemble your own image variant with your
-*packages lists* without having to compile everything yourself. The "OpenWrt-SDK" is
-the fastest way to build your own packages or programs without compiling OpenWrt itself.
-The "initrd" directory contains some initrd-images for netboot, which are required on
-some boards to initially install OpenWrt.
+As you notice there are several different image variants ("backbone", "default",
```

Comment:
See above: Should we use the changed packagelists here (notunnel, manual, ...)?

Reply:
included.